php-nginx/Dockerfile

```diff
@@ -4,13 +4,19 @@ MAINTAINER Kieren Evans <kieren.evans+cp-dockerfiles@inviqa.com>
 
 # Install PHP packages, including the Tideways extension
 ARG PHP_VERSION
-ENV PHP_VERSION=${PHP_VERSION:-7.0}
+ENV PHP_VERSION=${PHP_VERSION:-7.1}
```

Comment:
no need for this, please revert as 7.0 is LTS, and this doesn't even get the default used anyway